metrics/Issues_New.md

```diff
@@ -154,12 +154,17 @@ during a certain period (eg, a month).
 
 ## Known Implementations
 
-* [Grimoirelab](https://chaoss.github.io/grimoirelab). Enriched index for
-GitHub Issues, GitLab issues, Jira, Bugzilla, and
-Launchpad are composed of one item per issue,
-which makes it basically correspond to this metric when counted.
-The GitHub Issues, GitLab issues, Jira, Bugzilla, and
-Launchpad summary panels,
-available out of the box, provide exactly that.
+* [GrimoireLab](https://chaoss.github.io/grimoirelab) provides this metric out of the box for GitHub Issues, GitLab issues, Jira, Bugzilla, and Launchpad.
+  - View an example on the [CHAOSS instance of Bitergia Analytics](https://chaoss.biterg.io/app/kibana#/dashboard/GitHub-Issues).  
+  - Download and import a ready-to-go dashboard containing examples for this metric visualization from the [GrimoireLab Sigils panel collection](https://chaoss.github.io/grimoirelab-sigils/panels/github-issues/).
+  - Add a sample visualization to any GrimoreLab Kibiter dashboard following these instructions:
+    * Create a new `Vertical Bar` chart
+    * Select the `github_issues` index
+    * Filter: `pull_request` is `false`
+    * Metrics Y-axis: `Count` Aggregation, `# Issues` Custom Label
+    * Buckets X-axis: `Date Histogram` Aggregation, `grimoire_creation_date` Field, `Auto` Interval, `Time` Custom Label
+    * Buckets Split Series: `Terms` Sub Aggregation, `state` Field, `metric: # Issues` Order By, `Descending` Order, `1000` Size, `State` Custom Label
```

Comment:
It may be interesting noting that this provides a view of current state of the issues created at each corresponding time frame (bar).

Reply:
hmmm.... I don't understand your comment. @alpgarcia could you please elaborate?

Reply:
I think @alpgarcia refers to the fact that the bar chart, as described in the text, gives a list of new issues per time period (each bar). However, I think we can accept this as such, and if needed go through those details in a further iteration.